Deferred.podspec

```diff
@@ -0,0 +1,104 @@
+  #  summary should be tweet-length, and the description more in depth.
+  #
+
+  s.name         = "Deferred"
+  s.version      = "2.0b1"
+  s.summary      = "An implementation of OCaml's Deferred for Swift."
+
+  s.description  = <<-DESC
+  Deferred is an asynchronous promise-style API that can be used as an
+  alternative to the "block callback" pattern.
+                   DESC
+
+  s.homepage     = "https://github.com/bignerdranch/Deferred"
+
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See http://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  s.license      = "MIT"
+
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the authors of the library, with email addresses. Email addresses
+  #  of the authors are extracted from the SCM log. E.g. $ git log. CocoaPods also
+  #  accepts just a name if you'd rather not provide an email address.
+  #
+  #  Specify a social_media_url where others can refer to, for example a twitter
+  #  profile URL.
+  #
+
+  s.authors             = ["John Gallagher", "Zachary Waldowski"]
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  s.ios.deployment_target = "8.0"
+  s.osx.deployment_target = "10.10"
+
+
+  # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the location from where the source should be retrieved.
+  #  Supports git, hg, bzr, svn and HTTP.
+  #
+
+  s.source       = { :git => "https://github.com/bignerdranch/Deferred.git", :tag => "v2.0b1" }
+
+
+  # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  CocoaPods is smart about how it includes source code. For source files
+  #  giving a folder will include any swift, h, m, mm, c & cpp files.
+  #  For header files it will include any header in the folder.
+  #  Not including the public_header_files will make all headers public.
+  #
+
+  s.source_files  = "Deferred"
+
+  s.public_header_files = "Deferred/Deferred.h"
+  
+  # ――― Subspecs
+  
+  # s.subspec "iOS" do |iOS_spec|
+  #   iOS_spec.platforms = [:ios]
+  # end
+  #
+  # s.subspec "Mac" do |mac_spec|
+  #   mac_spec.platforms = [:osx]
+  # end
+
+  # ――― Project Settings ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If your library depends on compiler flags you can set them in the xcconfig hash
+  #  where they will only apply to your library. If you depend on other Podspecs
+  #  you can include multiple dependencies to ensure it works.
+
+  # s.requires_arc = true
```

Comment:
Certainly can't hurt, but I assumed that the nature of Swift would require it. Perhaps it's best to leave it on in the event some ObjC gets added? :shrug_emoji: